docker-compose.dev.local.yml

```diff
@@ -14,6 +14,7 @@ services:
       - ./data/logs:/home/hydra/log
       - ./scripts:/home/hydra/scripts
       - ./mnt:/mnt
+      - ./tmp/imports:/home/hydra/tmp/imports
```

Comment:
Is this what lets Sidekiq see the import files????

Reply:
Yes!! I have been manually loading sidekiq while debugging so it would see everything. Once I moved back to having sidekiq run automatically in its own container, it couldn't see the files. This should solve the issue!